CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# dbt_netsuite v0.19.0
+
+## Breaking Changes (full refresh required)
+- Added optional `fiscalcalendar` source table to support accurate fiscal year start dates (currently defaulted to calendar year). This table, related models (`stg_netsuite2__fiscal_calendar_tmp` and `stg_netsuite2__fiscal_calendar`), and relevant adjustments within `int_netsuite2__accounting_periods` are disabled by default. To enable this feature:
+  - Quickstart users: enable the fiscalcalendar table in the connection schema tab.
+  - dbt Core users: enable the fiscalcalendar table in the connection schema tab and also set the `netsuite2__fiscal_calendar_enabled` variable to true (default is false).
+
+## Under the Hood
```

Comment:
Should we call out the new macros we created?

Reply:
Good callout, would be relevant to add. Will update when I push the package dep change.